docs/editors/migration.md

```diff
@@ -1,34 +1,49 @@
 # Migrating from `ruff-lsp`
 
-While `ruff server` supports the same feature set as [`ruff-lsp`](https://github.com/astral-sh/ruff-lsp), migrating to
-`ruff server` may require changes to your Ruff or language server configuration.
+To provide some context, [`ruff-lsp`](https://github.com/astral-sh/ruff-lsp) is the LSP implementation for Ruff to power the editor
+integrations which is written in Python and is a separate package from Ruff itself. The **native
+server** is the LSP implementation which is written in Rust and is available under the `ruff server`
+command. This guide is intended to help users migrate from
+[`ruff-lsp`](https://github.com/astral-sh/ruff-lsp) to the native server.
 
 !!! note
 
-    The [VS Code extension](https://github.com/astral-sh/ruff-vscode) settings include documentation to indicate which
-    settings are supported by `ruff server`. As such, this migration guide is primarily targeted at editors that lack
-    explicit documentation for `ruff server` settings, such as Helix or Neovim.
+    The native server was first introduced in Ruff `0.3.5`, designated as beta in Ruff `0.4.5`, and
```

Comment:
The designated sounds a bit fancy here. How about

> The native server was first introduced in Ruff version 0.3.5. It was marked as beta in version 0.4.5 and officially stabilized in version 0.5.3.

or 

> The native server was introduced in Ruff 0.3.5, became beta in 0.4.5, and stabilized in 0.5.3.

Reply:
Updated to the first one that you've suggested.